### DISCUSSION TITLE
What exactly mean about the Walnut type in scvelo

### DISCUSSION THREAD

**Discussion by honghh2018** [theislab/scvelo#668] [Q&A]
Hi guys,
![image](https://user-images.githubusercontent.com/41668708/130747846-775ee756-36cb-46e5-b8c5-89373786ee42.png)
I had two genes like above plot and what exactly mean about picture that the red and yellow arrow point to, why the gap so difference between yellow arrow and the red arrow, what this condition mean ? 
 How can i explain this different?
Best,
Hanhuihong

**Comment by honghh2018** [comment]
Can any one hear this?

**Comment by WeilerP** [comment]
Sorry, not sure what you mean by walnut type and I don't understand your question(s).

I presume the arrows point at the inferred phase portrait but also not sure if this is what you were going for. The solid line is the simulated phase portrait, the dashed line the inferred steady state ratio.

**Reply by honghh2018** [reply]
Sorry to the ambigous,
     I can not explain those  pictures in two state of experiment i post above, what different in it? how can i explain the biological siginificant in this picture with same gene showing the different trend in picture?

Best
